### ISSUE TITLE
Cannot get the quickstart example to work

### ISSUE THREAD

**Issue by sivabudh** [lokra/seaweedfs-client#5]
This is the code that I have right now, and it puzzles me. It doesn't work. I'm sure I have SeaweedFS set up correctly because I got `pyseaweed` to work perfectly: https://github.com/utek/pyseaweed.
 
```
import org.lokra.seaweedfs.core.FileSource;
import org.lokra.seaweedfs.core.FileTemplate;

import java.io.*;

public class Hello {
    public static void main(String[] args) {
        System.out.println("Hello, World");

        FileSource fileSource = new FileSource();
        fileSource.setHost("localhost");
        fileSource.setPort(9333);
        try {
            fileSource.startup();
            System.out.println("Finished starting up!");
        } catch (IOException e) {
            System.out.println("Aiya! Cannot connect!");
            return;
        }

        try {
            InputStream is = new FileInputStream("/Users/Pac/Code/golang/flow.jpg");
            FileTemplate template = new FileTemplate(fileSource.getConnection());
            template.saveFileByStream("flow.jpg", is);
        } catch(FileNotFoundException e) {
            System.out.println("Cannot save the file!");
            return;
        } catch(IOException e) {
            System.out.println("Cannot write to Seaweed!");
            return;
        }

        System.out.println("Success and done!");
    }
}
```

This is the console output.

```
Hello, World
Nov 23, 2016 11:00:11 PM org.lokra.seaweedfs.core.FileSource startup
INFO: start connect to the seaweedfs core server [http://localhost:9333]
Nov 23, 2016 11:00:13 PM org.lokra.seaweedfs.core.Connection startup
INFO: core connection is startup now
Finished starting up!
329 [main] INFO org.ehcache.core.EhcacheManager - Cache 'lookupVolumeCache' created in EhcacheManager.
Cannot write to Seaweed!
```

**Comment by ChihoSin** [comment]
```
try {
            InputStream is = new FileInputStream("/Users/Pac/Code/golang/flow.jpg");
            FileTemplate template = new FileTemplate(fileSource.getConnection());
            template.saveFileByStream("flow.jpg", is);
}
```

Can you tell me which row is throw IOException.

**Comment by ChihoSin** [comment]
@sivabudh 
I fixed startup thread at version 0.7.3.RELEASE.
